### PR TITLE
[Update] Add Domain Shortguide

### PR DESCRIPTION
**Pull request by rsyracuse**
AXFR Update

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 5c49c12437d4e276ec2b04428cb9a682e19f531d

https://deploy-preview-3690--nostalgic-ptolemy-b01ab8.netlify.app

File: docs/platform/manager/dns-manager-add-domain-shortguide/index.md [hunk 1]
```diff
@@ -25,10 +25,9 @@ Creating a domain also creates its corresponding domain zone.
 
     {{< image src="add-new-domain.png" alt="This page lets you add a new domain" title="This page lets you add a new domain" >}}
 
-1. If you want to add a *slave zone* instead of a master zone, click the **Slave** radio button.
+1. If you want to add a *slave zone* instead of a master zone, click the **Slave** radio button. If not, you may skip to the next step.
 
-    {{< note >}}
-In order for Linode's DNS servers to function as slaves, your DNS master server must notify and allow AXFR requests from the following IP addresses:
+In order for Linode's DNS servers to function as slaves,your DNS master server must notify and allow AXFR requests from the following IP addresses:
```
Comment (andystevensname) [inline docs/platform/manager/dns-manager-add-domain-shortguide/index.md:30]: space between `slaves,` and `your`

Comment (andystevensname) [reply docs/platform/manager/dns-manager-add-domain-shortguide/index.md:30]: Looks like this Note isn't rendering for some reason: https://deploy-preview-3690--nostalgic-ptolemy-b01ab8.netlify.app/platform/manager/dns-manager/#add-a-domain

File: docs/platform/manager/dns-manager-add-domain-shortguide/index.md [hunk 2]
```diff
@@ -40,7 +39,10 @@ In order for Linode's DNS servers to function as slaves, your DNS master server
     2600:3c02::a
     2600:3c03::a
     2a01:7e00::a
-{{< /note >}}
+
+{{< caution >}}
+On **December 15th, 2020** the IP address `75.127.96.10` will be replaced with the IP address `74.207.225.10`. While both IP addresses currently provide AXFR support, users will need to replace `75.127.96.10` with `74.207.225.10` in their configurations by the 15th of December, 2020,for long term AXFR support.
```
Comment (andystevensname) [inline docs/platform/manager/dns-manager-add-domain-shortguide/index.md:44]: space between `2020,` and `for`

**Review by andystevensname — CHANGES_REQUESTED** [review]

**Pull request merged**